pkg/ctl/delete/delete.go.orig

```diff
@@ -0,0 +1,33 @@
+package delete
+
+import (
+	"github.com/kris-nova/logger"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+var (
+	waitDelete bool
+)
+
+// Command will create the `delete` commands
+func Command(g *cmdutils.Grouping) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "Delete resource(s)",
+		Run: func(c *cobra.Command, _ []string) {
+			if err := c.Help(); err != nil {
+				logger.Debug("ignoring error %q", err.Error())
+			}
+		},
+	}
+
+<<<<<<< HEAD
```

Comment:
This seems to have slipped-in, looks like the whole file was accidentally checked-in.